src/Features/Navigation/Init.php

```diff
@@ -16,14 +16,19 @@
  * Contains logic for the Navigation
  */
 class Init {
+	/**
+	 * Option name used to toggle this feature.
+	 */
+	const TOGGLE_OPTION_NAME = 'woocommerce_navigation_enabled';
```

Comment:
This is a tiny bit magic.  An alternative would be to store the toggleable features in the `Features` class as an associative array.

```php
$toggleable_features = array(
  'navigation' => 'woocommerce_navigation_enabled'
);
```

Though I do like the encapsulation using the current method.

Reply:
I'd lean a bit towards the associative array myself, just for clarity. As it is I wouldn't realize that refactoring/renaming `TOGGLE_OPTION_NAME` within a feature class would disrupt any functionality. 

Alternatively, we could rename that constant to something a little more descriptive, or even add a separate boolean flag that actually determines if the feature is disabled with tracking. ie:
```
const DISABLE_WITH_TRACKING = True;
const TOGGLE_OPTION_NAME = 'woocommerce_navigation_enabled';
```
That way the option name variable would just serve to indicate the option name, and the boolean would fairly clearly indicate the relationship with tracking.

Reply:
Thanks for the feedback here.  I'll update this to use the associative array; I think it does make things more clear.